datafusion/core/src/datasource/listing/table.rs

```diff
@@ -1898,6 +1898,9 @@ mod tests {
         Ok(())
     }
 
+    /// Note: We now default to use Utf8View, but we don't support for Utf8View in JSON reader
```

Comment:
This testing failed with error JSON reader don't support utf8view, we may need to file a ticket for arrow-rs?

Reply:
I think this is the risk for the PR @alamb

Reply:
- The follow on ticket is here: https://github.com/apache/arrow-rs/issues/7244